recipes/app1.rb

```diff
@@ -18,23 +18,27 @@
 
 include_recipe 'osl-app::default'
 
-node.normal['users'] = %w(openid-staging openid-production)
+node.default['users'] = %w(openid-staging openid-production)
 
 openid_secrets = data_bag_item('osl-app', 'openid')
 
 #### Sudo Privs ####
 
+# fails idempotency
 sudo 'openid-staging' do
   user 'openid-staging'
   commands sudo_commands('openid-staging-unicorn', 'openid-staging-delayed-job')
   nopasswd true
+  not_if 'getent passwd openid-staging'
```

Comment:
This is no longer needed now that ``node.default`` is being used above. Can you please remove all references to this change globally?